Crashlytics/Crashlytics/Models/Record/FIRCLSRecordMachException.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import "FIRCLSRecordCrashBase.h"
+#import "FIRCLSRecordFrame.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// One way to trigger Mach exceptions is by accessing un-malloc'd memory
+@interface FIRCLSRecordMachException : FIRCLSRecordCrashBase
+
+@property(nonatomic, copy) NSMutableArray<NSNumber *> *codes;
```

Comment:
The underlying type can be a NSMutableArray but for the header this should be NSArray